Modules/Core/Common/include/itkBoundingBox.h

```diff
@@ -89,7 +89,7 @@ class ITK_TEMPLATE_EXPORT BoundingBox : public Object
 
   /** Hold on to the type information specified by the template parameters. */
   using PointIdentifier = TPointIdentifier;
-  using CoordRepType = TCoordRep;
+  using CoordRepType = TCoordinate;
```

Comment:
This is a bit funny, but as it is part of the public interface it is better not to change it. Too many headaches for little gain.

Reply:
@dzenanz Indeed 🤔  I'm considering a follow-up PR that would add lines like `using CoordinateType = TCoordRep`. But I understand that the old `CoordRepType` aliases must still be preserved for quite a while.